slack-base/src/main/java/com/hubspot/slack/client/models/interaction/ShortcutIF.java

```diff
@@ -0,0 +1,18 @@
+package com.hubspot.slack.client.models.interaction;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.SlackChannel;
+import org.immutables.value.Value.Immutable;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ShortcutIF extends SlackInteractiveCallback {
+  String getTriggerId();
+
+  default SlackChannel getChannel() {
```

Comment:
This feels like it should be `Optional<SlackChannel>` to capture that we're dealing with something that may not be present (otherwise folks will have to handle null in their code, which is an explicit antipattern

Reply:
The reason I didn't make that change was because it would require changing `SlackInteractiveCallback` which seemed like a big breaking change to the API affecting a lot of users.

If you're good with that, I think I should probably resubmit as a separate PR dedicated for just this change because it's going to impact a lot of files.